src/server/stream_family.cc

```diff
@@ -543,11 +543,11 @@ OpResult<streamID> OpAdd(const OpArgs& op_args, string_view key, const AddOpts&
       streamTrimByID(stream_inst, opts.minid.val, opts.trim_approx);
     }
   } else {
-    streamAddTrimArgs add_args = {
-        .trim_strategy = static_cast<int>(opts.trim_strategy),
-        .approx_trim = opts.trim_approx,
-        .limit = opts.limit,
-    };
+    streamAddTrimArgs add_args;
```

Comment:
Ok, but I am unsure - doesn't C++ automatically set struct field to empty values?

Reply:
It's complicated... to be technical, the trivial default constructor leaves integer members uninitialized. Adding '= {0}' makes it into a zero initialization so you get all zeros instead of random stack memory.

(obligatory link to https://twitter.com/i/status/1004017362381795329)

Reply:
done :-)